dashboard/lib/routes/cbroker.js

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2015 Telefónica I+D
+ * All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may obtain
+ * a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+'use strict';
+
+var http = require('http'),
+    cbroker = {};
+
+
+var FIELDNAME = 'sanity_status'; // field name for value about regions status
+
+
+/**
+ * @function parseRegions
+ *
+ * @param {Object} entities
+ * @return {Array}
+ */
+cbroker.parseRegions = function (entities) {
+
```

Comment:
This is more a private function than a method. Why not define it as ``function parseRegions(entities)``?